tests/integration/cmd_namespace_test.go

```diff
@@ -2,8 +2,9 @@ package integration
 
 import (
 	"fmt"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
```

Comment:
Non stdlib imports should not appear in first block of imports

Reply:
https://github.com/redhat-developer/odo/pull/6166#discussion_r985388275